### PR TITLE
Add: Interface

### PR DESCRIPTION
**Pull request by lukman155**
Use React Router.
Use styles in a React app.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ function Calculator() {
   const { total, next } = state;
 
   return (
-    <CalculatorContainer>
+    <CalculatorContainer className="calculator-container">
       <Display>
```
Comment (zieeco) [inline src/components/Calculator.js:18]: - Hey, great job with the styling, however, let's kindly follow the layout of the design as much as we can. let's kindly align  this section to match the template below:



*yo8ur design*
Screenshot from 2022-12-07 01-33-30

*templeate*
Screenshot from 2022-12-07 01-33-04

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @lukman155 ,


Great job so far!  👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (lukman155) [reply src/components/Calculator.js:18]: Hello, @zieeco thank you for your kind words. Side by side isn't as pretty but I'll implement it.

Comment (peterrobert) [reply src/components/Calculator.js:18]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @lukman155, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**